documentation/physics-models/plasma_radiation.md

```diff
@@ -30,8 +30,19 @@ $$
 P_i = n_i n_e L_Z (Z_i, T)
 $$
 
-where \(P_i\) is the radiation per unit volume (excluding synchrotron radiation),
-\(L_Z (Z_i, T)\) is the loss function for ion species \(i\) at temperature \(T\), and \(n_i\) is the density of ion species \(i\).
+where $P_i$ is the radiation per unit volume (excluding synchrotron radiation),
+$L_Z (Z_i, T)$ is the loss function for ion species $i$ at temperature $T$, and $n_i$ is the density of ion species $i$.
+
+!!! note "Particle confinement"
+    For the loss function $L_Z (Z_i, T)$, the particles are assumed to be in equailibrium and thus have infinite confinement time. (This is the data found at the bottom of the ADAS derived files in `process/data/lz_non_corona_14_elements`)
```

Comment:
Is it really the case that they have infinite confinement time? Radiating impurities are "just there", and aren't being added/lost? Maybe be explicit that by equilibrium we mean "no transport". You kind of say this below though, now I see it...